lib/puppet_x/cisco/transport_shim.rb

```diff
@@ -0,0 +1,342 @@
+require 'hocon'
```

Comment:
Should the filepath be lib/puppet_x/cisco_nexus/transport_shim.rb ?

Reply:
Moved